Resources/template.json

```diff
@@ -274,7 +274,15 @@
               "get",
               "list"
             ]
-          }
+          },
+          "certificates": [
+            "get",
+            "list",
+            "create",

```

Comment:
Can you remove create and update?  This resource takes the ids to grant permissions to from a parameter called defaultKeyVaultReadAccess.  If we need to grant other rights by default I think this should be configured via a separate resource